include/gbinder_types.h

```diff
@@ -140,10 +141,6 @@ typedef struct gbinder_hidl_memory {
     GBinderHidlString name;
 } GBinderHidlMemory; /* Since 1.1.4 */
 
-#define GBINDER_HIDL_MEMORY_PTR_OFFSET (0)
-#define GBINDER_HIDL_MEMORY_NAME_OFFSET (24)
-G_STATIC_ASSERT(sizeof(GBinderHidlMemory) == 40);
-
```

Comment:
This should stay here, for two reasons:
1. It may break compilation if something relies on these constants being defined in `gbinder_types.h`
2. Closely related things should be close to each other unless there's a good reason to move them apart.

Reply:
This was moved accidentally. I moved it back.